srml/democracy/src/lib.rs

```diff
@@ -349,6 +432,11 @@ impl<T: Trait> Module<T> {
 			})
 	}
 
+	/// # <weight>
+	/// - Enumerates over the delegations held in storage.
+	/// - Recursively calls `delegated_votes`
+	/// - Special attention to `recursion_limit` to prevent too many recursive calls.
```

Comment:
Remove line above and modify this one:
```suggestion
	/// - Recursive function with recursion limit set as module constant.
```